File: src/main/java/com/sf_lolitahag/action/pitchs/Fast.java
/*
Copyright (C) 2016 Koji Wakamiya
Copyright (C) 2016 Masanori Kojima
Released under the MIT license
http://opensource.org/licenses/mit-license.php
*/
package com.sf_lolitahag.action.pitchs;

public class Fast implements IPitch {

  private static final int BALL_SPEED = 40;

  @Override
  public int getUpdateX(int currentX) {
    return NONE;
  }

  @Override
  public int getUpdateY(int currentY) {
    return BALL_SPEED;
  }

  @Override
  public boolean isSpin() {
    return false;
  }
}

File: src/main/kotlin/com/sf_lolitahag/pitchs/Fast.kt
/*
Copyright (C) 2016 Koji Wakamiya
Copyright (C) 2016 Masanori Kojima
Released under the MIT license
http://opensource.org/licenses/mit-license.php
*/
package com.sf_lolitahag.pitchs

class Fast : IPitch {

    private val BALL_SPEED = 40

    override fun getUpdateX(currentX: Int): Int = IPitch.NONE

    override fun getUpdateY(currentY: Int): Int = BALL_SPEED

    override val isSpin: Boolean
        get() = false
}
